composer.json

```diff
@@ -25,7 +25,7 @@
     }
   ],
   "require": {
-    "php": "^7.2 || ^8.0",
+    "php": "^7.2 || ^8.3",
```

Comment:
Tímhle říkáš že to může běžet jenom na 8.3 a vyšší, takže by to třeba nemohlo být na 8.2, to je chtěné?

Reply:
díky spajxo, máš pravdu, že zápis byl validní, problém bude jinde